packages/backend/src/database/entities/dashboards.ts

```diff
@@ -98,3 +99,16 @@ type DbDashboardTileMarkdowns = {
 
 export type DashboardTileMarkdownsTable =
     Knex.CompositeTableType<DbDashboardTileMarkdowns>;
+
+export type DbDashboardTileComments = {
```

Comment:
are you moving this away from `dashboards`  ? I need to update it , so if you're going to move it I can wait to avoid a conflict

Reply:
also I'd call it just `DbComments` , but this is a personal opinion. I'm bad at naming so don't trust me

Reply:
Good point, I will move to `/entities/comments.ts` and keep the naming to match the table name 👍